ui/01-is.yaml

```
@@ -0,0 +1,27 @@
+  lookupPolicy:
+    local: false
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: python-312
+spec:
+  lookupPolicy:
+    local: false
+  tags:
+  - annotations: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)